WordPress/src/main/res/layout/person_detail_fragment.xml

```diff
@@ -1,97 +1,63 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
     SlidingRelativeLayout is used for animating the slide in-out between fragments
-    Background color is set so the fragment below it, PeopleListFragment won't be visible
+    Background color is set so the fragment below it, PeopleListFragment, won't be visible
 -->
 <org.wordpress.android.ui.SlidingRelativeLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/default_background"
-    android:orientation="vertical">
+    android:background="@color/white"
+    android:clickable="true">
 
-    <RelativeLayout
+    <include
         android:id="@+id/person_details"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content">
+        layout="@layout/person_detail"/>
 
-        <org.wordpress.android.widgets.WPNetworkImageView
-            android:id="@+id/person_avatar"
-            android:layout_width="@dimen/avatar_sz_large"
-            android:layout_height="@dimen/avatar_sz_large"
-            android:layout_marginLeft="@dimen/margin_extra_large"
-            android:layout_marginTop="@dimen/margin_extra_large" />
+    <LinearLayout
```

Comment:
It seems that the role elements have exactly the same styling as the person's name ones but, in practice, the role row is tappable. I was a bit confused when I first reached this screen as I didn't have a a clue that I could change the role here. After I found out, I tried to tap the username row to see what will happen (which ofc didn't do something).

So, there seems to be a minor design issue here. I don't consider it a blocker for the PR but perhaps we can improve it. Cheers!